### PR TITLE
Documentation Coverage by sphinx.ext.coverage #519

### PR DESCRIPTION
**Pull request by tkoyama010**
Subject: Documentation Coverage by sphinx.ext.coverage #519

### Feature or Bugfix
Feature

### Purpose
We can see the coverage like this.
https://travis-ci.com/tkoyama010/pyvista/jobs/285757449#L959-L964

### CODE REVIEW

**Comment by tkoyama010** [comment]
I made a PR of #519 by sphinx but it may not be what we want :confused:

**Comment by akaszynski** [comment]
@banesullivan, could you please take a look at this since as you requested the doc coverage in #519?

**Comment by banesullivan** [comment]
I tried this out locally and I think this is working well! Locally from master, I see the following:

```
Undocumented Python objects
===========================
Modules that failed to import
-----------------------------
 * pyvista.composite -- No module named 'pyvista.composite'
 * pyvista.filters -- No module named 'pyvista.filters'
```

it would be nice if we could incorporate this into the codestyle/codespell status check

@tkoyama010: when you have a chance, would you please update this branch from `master` and maybe include what is needed in the Azure script since we have abandoned Travis?

**Comment by tkoyama010** [comment]
@banesullivan Thanks for your review. I'm glad the change helped. Please wait while I add changes.

**Comment by tkoyama010** [comment]
Sorry I'm not good at Azure script. Does anyone know what is a best way to solve this?

**Comment by akaszynski** [comment]
You can get a log of the build script by clicking on Details and then "View more details on Azure Pipelines".  The final link is [pyvista pipeline](https://dev.azure.com/pyvista/53bd8a97-5c2b-477c-b85e-e3a206c839a7/_build/results?buildId=2133&view=logs&jobId=0d0124fa-d3a8-51e8-d2f8-a0118e38d729).  From there you can view the log and see why it errored out.

This bug is occurring on the BuildDocumentation step.  You can run this locally with:
`make -C docs html` from the root `pyvista` directory.  I'm not sure how it's run on Windows, but if you look up the docs on [sphinx-doc](https://www.sphinx-doc.org/tutorial.html) you can just build it without using `make`.

**Comment by banesullivan** [comment]
Looks like the path is just incorrect:

```
Testing of coverage in the sources finished, look at the results in _build/html/python.txt.

Sphinx-gallery successfully executed 66 out of 66 files subselected by:

    gallery_conf["filename_pattern"] = '\\.py'
    gallery_conf["ignore_pattern"]   = '__init__\\.py'

after excluding 0 files that had previously been run (based on MD5).

make: Leaving directory '/home/vsts/work/1/s/docs'
cat: doc/_build/html/python.txt: No such file or directory
```

should be `docs/` not `doc/`

**Comment by tkoyama010** [comment]
Thanks

File: azure-pipelines.yml [hunk 1]
```diff
@@ -253,6 +253,7 @@ jobs:
 
   - script: |
       make -C docs html
+      cat doc/_build/html/python.txt
```
Comment (tkoyama010) [inline azure-pipelines.yml:256]:
```suggestion
      cat docs/_build/html/python.txt
```

**Pull request merged**